content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -61,7 +61,7 @@ operation.
   dependencies and outputs, to allow safely running multiple DVC commands in
   parallel
 
-- `.dvc/tmp/exp<n>`:  _n_<sup>th</sup> workspace copy, used for temporary or
+- `.dvc/tmp/exp<n>`: _n_<sup>th</sup> workspace copy, used for temporary or
```

Comment:
Hmmm this passed my local tests 🤷

Reply:
Ah no it didn't. I just forgot to commit that change from `prettier`...

Reply:
But why is it passing when the unstaged changes are correct? It should check only staged changes!